ols/app/models/models.py

```diff
@@ -283,6 +283,14 @@ def check_uuid(cls, value: str) -> str:
             raise ValueError(f"Improper conversation ID {value}")
         return value
 
+    @field_validator("sentiment")
+    @classmethod
+    def check_sentiment(cls, value: int) -> int:
```

Comment:
is `None` allowed?

Reply:
so it's `Optional[int]` -> `Optional[int]`?

Reply:
I mean, just specify that the member in model is optional, the parameter is optional, and indeed, return value is optional too ;)

Reply:
Ok, you are talking about typing. Updated.